src/components/PageNotFound.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+
+
+const PageNotFound = () => {
+    return (
+        <div className="container-fluid">
+        <div className="text-center">
+        <img src="img/footer/404.png" width="500" alt="404"/>  
```

Comment:
Move this to 404 folder in images directory.

Reply:
done

Reply:
👍